apps/web/test/utils/bookingScenario/expects.ts

```diff
@@ -304,8 +304,41 @@ export function expectWebhookToHaveBeenCalledWith(
   }
 }
 
-export function expectWorkflowToBeTriggered() {
-  // TODO: Implement this.
+export function expectWorkflowToBeTriggered({
+  emails,
+  organizer,
+}: {
+  emails: Fixtures["emails"];
+  organizer: { email: string; name: string; timeZone: string };
+}) {
+  const subjectPattern = /^Reminder: /i;
```

Comment:
For now I am only testing workflows with reminder template, so subject will always start with 'Reminder: '